dist/PublicLab.Editor.js

```diff
@@ -21778,6 +21779,10 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
       progressall: function (e, data) {
 
         var progress = parseInt(data.loaded / data.total * 100, 10);
+        
```

Comment:
Hmm, maybe this needs to be rebuilt without unrelated changes?

Reply:
I don't know how these changes go added here, but I have just resolved it.